tools/cdb2sql/cdb2sql.cpp

```diff
@@ -1589,6 +1589,13 @@ static int run_statement(const char *sql, int ntypes, int *types,
             printf("Effects not sent by comdb2 server. \n");
         }
     }
+
+    if (docost) {
```

Comment:
We need to use the same checks here. We probably do not want to print cost if there's a bad rcode. We do not want to print cost twice if a custom `report_costs` is defined.

Reply:
We don't get here in rc is not OK_DONE (line 1574).